packages/framework/src/Controller/Admin/ProductController.php

```diff
@@ -423,4 +451,30 @@ protected function productCanBeCreated()
 
         return true;
     }
+
+    /**
+     * @internal Will be removed in the next major release
+     */
+    protected function validateInjectedDependencies(): void
```

Comment:
This should be part of the `ProductSearchExportScheduler`

Reply:
Actually, it shouldn't because I need to know if the dependency was properly injected into `ProductController`.